### DISCUSSION TITLE
Error while installing ipa through AltStore on iOS 17.2.1

### DISCUSSION THREAD

**Discussion by hai5ive** [KRTirtho/spotube#1037] [Q&A] [answered]
Wanted to test the iOS ipa and getting the following error while installing with AltStore:

> AltSign.Error 0 /Users/rileytestut/Developer/Projects/AltStore/Dependencies/AltSign/AltSign/ldid/.. /.. /Dependencies/ldid/ldid.cpp(1368): _assert(): end >= size - 0x10

Downloaded the ipa from spottune/nightly as well as powenns 3.4.0 dev version

iOs 17.2.1

Couldn't find a solution. Other apps install well. Any recomondations how to get it working?

**Comment by KRTirtho** [comment]
@powenn Can you help?

**Reply by powenn** [reply, marked as answer]
Try sideloadly, I remembered someone said he got ipa installed with sideloadly

**Reply by hai5ive** [reply]
Thanks, it worked with sideloadly. Sadly, I can't use autosign because of arm.

I noticed that I can't use the music playback (so put the app on background or lock the phone). Is this a known issue, or am I doing something wrong?

**Reply by powenn** [reply]
> Thanks, it worked with sideloadly. Sadly, I can't use autosign because of arm.
> 
> 
> 
> I noticed that I can't use the music playback (so put the app on background or lock the phone). Is this a known issue, or am I doing something wrong?

Does Spotube support this feature?

**Reply by mehulparmar4ever** [reply]
Thanks, it worked with sideloadly. Installation done. app works fine in iOS.
_Unable to download songs in iOS 17.6.1 apk v3.8.0 stable build v33_
